bigframes/bigquery/__init__.py

```diff
@@ -119,3 +118,33 @@ def array_agg(
         raise ValueError(
             f"Unsupported type {type(obj)} to apply `array_agg` function. {constants.FEEDBACK_LINK}"
         )
+
+
+def array_to_string(series: series.Series, delimiter: str) -> series.Series:
```

Comment:
Should we add `null_text` as an optional parameter, too?

https://cloud.google.com/bigquery/docs/reference/standard-sql/array_functions#array_to_string

I recall it's a bit difficult to test arrays containing nulls since there are many paths that don't support them, so I'm okay omitting for now.

Reply:
I initially experimented with using `null_text` but determined it wasn't beneficial at this stage. The `null_text` parameter is designed to handle null values within arrays, whereas BF arrays cannot contain nulls. This stems from BigQuery's behavior of permitting nulls in intermediate query results but not in materialized views or tables.